docs/pages/guides/open-frames.mdx

```diff
@@ -50,4 +50,21 @@ const handleRequest = frames(async (ctx) => {
 });
 ```
 
+## Unauthenticated Frames
+
+Open Frames can be used to create unauthenticated Frames that can be supported by all clients by using the anonymous frames convention, which is built into the openframes middleware.
+
+In order to support unauthenticated frames in your application, simply include `openframes()` without supplying any arguments in your `createFrames` middleware.
+
+```tsx
+const frames = createFrames({
+  // ...
+  middleware: [
+    openframes(),
```

Comment:
I think it'd help to have an example of multiple client protocols on this page. Currently somewhat unclear how to combine this

Reply:
oops, missed this. will update